torch/onnx/_internal/exporter/_building.py

```diff
@@ -465,6 +465,12 @@ def _construct_node(
         else:
             inputs.append(value)
 
+    # If final inputs are None, strip them in the graph
+    for input in reversed(inputs):
+        if input is not None:
+            break
+        inputs.pop()
```

Comment:
Not sure if python will complain when modifying the list while iterating on reverse?

Reply:
Just checked. This should be fine:

```python
>>> a = [1,2,3,3]
>>> for i in reversed(a):
...     if i == 3: a.pop()
... 
3
3
>>> a
[1, 2]
```